http/http-api/pom.xml

```diff
@@ -59,6 +59,11 @@
             <artifactId>logback-classic</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.okta.sdk</groupId>
+            <artifactId>okta-sdk-impl</artifactId>
+            <version>3.0.1</version>
+        </dependency>
```

Comment:
This creates a circular dependency.  The "commons" project cannot depend on the SDK

Reply:
fixed